starter-code/index.js

```diff
@@ -1,11 +1,32 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+
+
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort((a, b) => a - b);
+  }
+
+  get(pos) { return this.items[pos - 1]; }
+
+  max() { return this.items[this.length - 1]; }
```

Comment:
faltou incluir
`if (this.length === 0){
    throw new Error('EmptySortedList');
}`